systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlConfigurationST.java

```diff
@@ -63,38 +65,87 @@ public class CruiseControlConfigurationST extends AbstractST {
 
     private final String namespace = testSuiteNamespaceManager.getMapOfAdditionalNamespaces().get(CruiseControlConfigurationST.class.getSimpleName()).stream().findFirst().get();
 
+    private static Double removeKibSuffix(String s) {
+        return Double.valueOf(s.substring(0, s.length() - 5));
+    }
+
     @ParallelNamespaceTest
     void testCapacityFile(ExtensionContext extensionContext) {
         final String namespaceName = StUtils.getNamespaceBasedOnRbac(namespace, extensionContext);
         final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
 
-        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaWithCruiseControl(clusterName, 3, 3).build());
+        double disk = 100_000;
+        double cpu = 100;
+        String inboundNetwork = "10000KiB/s";
+        String outboundNetwork = "10000KiB/s";
+        String inboundNetworkOverride0 = "20000KiB/s";
+        String inboundNetworkOverride1 = "40000KiB/s";
+        String outboundNetworkOverride1 = "400000KiB/s";
+
+        Integer broker0 = 0;
+        Integer broker1 = 1;
+        Integer broker2 = 2;
```

Comment:
I guess we can go with primitive `int` here - or maybe pass it directly to the list